moralpain/lib/submitted/submitted_route.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+class SubmittedRoute extends StatelessWidget {
+  const SubmittedRoute({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: const Color(0xFF232D4B),
+        title: const Text('Thank you, your response has been recorded. '),
+      ),
+      body: Center(
+        child: ElevatedButton(
```

Comment:
We'll want to get Vanessa's opinion about exactly what should be on this page. A button was a good first go.